File: C# Part 2/03.Methods/02.GetLargestNumber.cs
using System;
using System.Linq;

namespace GetLargestNumber
{
    internal class GetLargestNumber
    {
        private static void Main()
        {
            int max = int.MinValue;
            int[] input = Console.ReadLine().Split(' ').Select(x => Convert.ToInt32(x)).ToArray();
            foreach (var num in input) max = GetMax(max, num);

            Console.WriteLine(max);
        }
        public static int GetMax(int valA, int valB) => Math.Max(valA, valB);
    }
}

File: JavaScript Fundamentals/08. Functions/02_GetLargestNumber.js
function solve(args) {
    var input = args[0].split(' ');

    function GetMax(a, b) {
        if (a > b) {
            return a;
        } else if (a < b) {
            return b;
        } else {
            return a;
        }
    }

    var output = GetMax(+input[0], GetMax(+input[1], +input[2]));
    console.log(output);
}

solve(['8 3 6'])





